package.json

```diff
@@ -146,6 +147,8 @@
     "react-dom-factories": "^1.0.2",
     "react-sizeme": "^2.6.7",
     "slate": "^0.47.8",
+    "slate-html-serializer": "^0.8.10",
+    "slate-md-serializer": "^5.4.3-1",
```

Comment:
Out of curiosity, did you have any difficulty getting this to work? In my (brief) experiment I ran into a problem -- I can't remember exactly what it was -- that I thought was related to lack of support for recent changes to the Slate data model and ended up using the `@edithq/slate-md-serializer` fork instead. If this works, I'm happy to use it, but it's worth remembering that there are other forks out there if we ever have any problems.

Reply:
I did have some trouble -- partly due to the specifics of the serializer's implementation and partly due to my learning curve with Slate over the last couple of weeks. I actually tried out more than one serializer approach/package before picking this one.

* `slate-md-serializer` (the one I eventually used)
* `@scottage/slate-markdown-serializer`
* `markdown-serializer` by digitalhydra on GitHub.

The most complete one is the third one, above,  but it is considerably more complex and has no documentation to speak of -- so it's a lot of code reading.

The second one, above, is a small evolution of the "slate semi-official" one. It could be made to work, but it handles paragraphs a little differently and would need to have at least one extra rule to be defined to get to this current stage of functionality. And I just 

So, I ended up sticking with the easiest path. If we were to get much fancier, than I think we should probably fork the best one (digitalhydra's) and augment it have a swappable mapping of mark/block names and we'd have all our bases covered. But that's probably too time costly.